app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -74,18 +74,15 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
           setImmediate(() => {
             // continue with navigation
             e.target.disabled = false;
-            e.target.click();
             trackEvent('IdV: location submitted', {
               selected_location: selectedLocationAddress,
               in_person_cta_variant: inPersonCtaVariantActive,
             });
+            forceRedirect(inPersonURL!);
             // allow process to be re-triggered in case submission did not work as expected
             setAutoSubmit(false);
           });
-          forceRedirect(inPersonURL!);
         }
-      } catch {
-        setAutoSubmit(false);
```

Comment:
This line should very likely remain here. This is here to help the user try again if needed.